README.md

```diff
@@ -225,13 +261,173 @@ export async function getStaticProps({ preview = false }) {
+In `_app/page.jsx`, use it as follows:
+
+```js
+import { getStoryblokApi, StoryblokComponent } from "@storyblok/react/rsc";
+
+export default async function Home() {
+  const { data } = await fetchData();
+
+  return (
+    <div>
+      <h1>Story: {data.story.id}</h1>
+      <StoryblokComponent blok={data.story.content} />
+    </div>
+  );
+}
+
+export async function fetchData() {
+  const storyblokApi = getStoryblokApi();
+  return storyblokApi.get(`cdn/stories/home`, { version: "draft" });
+}
+```
+
+`StoryblokComponent` here renders all the components dynamically which you loaded before during the initalization inside the `storyblokInit` function. 
+
+**Demo: see _playground-next13-rsc_**
+
+## Next 13 (with Live Editing)
+
+You can enable the live editing with Server Side Components by rendering them with a wrapper (`StoryblokPovider`) on client. The SDK allows you to take full advantage of the Live Editing, but the use of Server Side Components is partial, which is still better than Next.js 12 performance wise. 
+
+> The SDK has a special module for RSC. Always use `@storyblok/react/rsc` for importing while using Server Side Components.
+
+### 1. Initialize
+
+In `_app/layout.jsx`, call `storyblokInit` without loading the components and use `StoryblokProvider` (this provider is created in the next step) :
+
+```js
+import { storyblokInit, apiPlugin } from "@storyblok/react/rsc";
+import StoryblokProvider from "../components/StoryblokProvider";
+
+storyblokInit({
+  accessToken: "OurklwV5XsDJTIE1NJaD2wtt",
+  use: [apiPlugin],
+});
+
+export default function RootLayout({ children }) {
+  return (
+    <StoryblokProvider>
+      <html lang="en">
+        <body>{children}</body>
+      </html>
+    </StoryblokProvider>
+  );
+}
+```
+
+### 2. Create StoryblokProvider and Import your Storyblok Components
+
+Create `components/StoryblokProvider.jsx`, and re-initalize and import your Storyblok components:
+
+```js
+/** 1. Tag it as client component */
+"use client";
+import { storyblokInit, apiPlugin } from "@storyblok/react/rsc";
+
+/** 2. Import your components */
+import Page from "../components/Page";
+import Teaser from "../components/Teaser";
+
+/** 3. Initialize it as usual */
+storyblokInit({
+  accessToken: "OurklwV5XsDJTIE1NJaD2wtt",
+  use: [apiPlugin],
+  components: {
+    teaser: Teaser,
+    page: Page,
+  },
+});
+
+export default function StoryblokProvider({ children }) {
+  return children;
+}
+```
+
+> Note: it's necessary to re-initialize here as well, as to enable the live editing you must initialize the lib universally (client + server) via a React client component.
+
+### 3. Fetch Content and Render Components
+
+The `getStoryblokApi` can be used to fetch the data from Storyblok. This is imported from `@storyblok/react/rsc`. Instead of using `StoryblokCompnent`, use `StoryblokStory` which will automatically handle the live event of the story. In `_app/page.jsx`, use them as follows:
```

Comment:
```suggestion
The `getStoryblokApi` can be used to fetch the data from the Storyblok API. This should be imported from `@storyblok/react/rsc`. 
Instead of using `StoryblokCompnent`, you can render the content of your route with the `StoryblokStory` component, which will automatically handle the Visual Editor live events when editing the story. In `app/page.jsx`, use them as follows:
```